src/ta/common.rs

```diff
@@ -553,6 +556,8 @@ impl fmt::Display for TrustAnchorSignedRequest {
 pub struct TrustAnchorSignerRequest {
     pub nonce: Nonce, // should be matched in response (replay protection)
     pub child_requests: Vec<TrustAnchorChildRequests>,
+    pub timing: TaTimingConfig,
```

Comment:
Would it make sense to make this make this field optional and then allow both this one and renew_time to be absent for compatibility?